src/modules/sampleread/SampleReadEventHandler.ts

```diff
@@ -64,31 +67,28 @@ export default class SampleReadEventHandler {
 
     // modern Chrome requires { passive: false } when adding event
     let supportsPassive = false;
-    try {
-      window.addEventListener(
-        "test",
-        null,
-        Object.defineProperty({}, "passive", {
-          get: function () {
-            supportsPassive = true;
-          },
-        })
-      );
-    } catch (e) {}
+
+    let opts =
+      Object.defineProperty &&
+      Object.defineProperty({}, "passive", {
+        // eslint-disable-next-line getter-return
+        get: function () {
+          supportsPassive = true;
+        },
+      });
+    window.addEventListener("test", function () {}, opts);
```

Comment:
Is this meant to be here?

Reply:
yes it is, it's a refactor from the lines above, after turning strict null check on, some of the syntax from before was acting up